hackathon_site/hardware/test_api.py

```diff
@@ -864,6 +864,89 @@ def test_successful_post(self):
             self.assertEqual(final_response[attribute], self.request_data[attribute])
 
 
+class IncidentDetailViewGetTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.profile = Profile.objects.create(user=self.user)
```

Comment:
you're testing an admin endpoint, so a user will not have a profile, you can remove this.

Reply:
similarly, admins will not have teams so you can remove line 871 as well

Reply:
Doesn't the Order object require a team, so wouldn't I keep line 871? With the team it passes the test cases.

Reply:
Ah true, keep it then.